third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py

```diff
@@ -9,8 +9,10 @@
 def _get_fk_table_name(column):
 	for fk in column.foreign_keys:
 		colspec = fk._get_colspec()
-		tablename, _ = colspec.split(".")
-		return tablename
+		splitted = colspec.split(".")
+		assert len(splitted) >= 2
```

Comment:
will the length ever be > 2?

Reply:
Yes, it can be `schemaname`.`tablename`.`columnname`.